locale/es.po

```diff
@@ -976,7 +976,7 @@ msgstr "Paridad impar no soportada"
 
 #: ports/atmel-samd/common-hal/audiobusio/PDMIn.c
 msgid "Only 8 or 16 bit mono with "
-msgstr "Solo mono de 8 o 16 bit con "
+msgstr "Solo mono de 8 ó 16 bit con "
```

Comment:
"Only 8 or 16 bit mono with " MP_STRINGIFY(OVERSAMPLING) "x oversampling is supported."
"Solo 8 ó 16 bit mono con " MP_STRINGIFY(OVERSAMPLING) "x de sobremuestreo es soportado."